packages/block-editor/src/components/autocomplete/index.js

```diff
@@ -59,6 +60,7 @@ export function useBlockEditorAutocompleteProps( props ) {
 	return useAutocompleteProps( {
 		...props,
 		completers: useCompleters( props ),
+		__unstableAccessibilitySlotName: ACCESSIBILITY_SLOT_NAME,
```

Comment:
I used slot/fill for now but I may be able to create a cleaner solution without passing additional props: check if the contentRef is in a different document and then create a portal mounting at the body.